packages/camera/camera_android/android/src/test/java/io/flutter/plugins/camera/features/exposurepoint/ExposurePointFeatureTest.java

```diff
@@ -308,6 +310,28 @@ public void updateBuilder_shouldNotSetMeteringRectangleWhenNoValidCoordsAreSuppl
     exposurePointFeature.updateBuilder(mockCaptureRequestBuilder);
     exposurePointFeature.setValue(new Point(null, 0d));
     exposurePointFeature.updateBuilder(mockCaptureRequestBuilder);
-    verify(mockCaptureRequestBuilder, times(3)).set(any(), isNull());
+    verify(mockCaptureRequestBuilder, times(3)).set(any(), any());
+  }
+
+  @Test
+  public void updateBuilder_shouldSetNonNullMeteringRectangleWhenNullValueIsSupplied() {
+    CameraProperties mockCameraProperties = mock(CameraProperties.class);
+    when(mockCameraProperties.getControlMaxRegionsAutoExposure()).thenReturn(1);
+    CaptureRequest.Builder mockCaptureRequestBuilder = mock(CaptureRequest.Builder.class);
+    ExposurePointFeature exposurePointFeature =
+        new ExposurePointFeature(mockCameraProperties, mockSensorOrientationFeature);
+
+    exposurePointFeature.setCameraBoundaries(this.mockCameraBoundaries);
+
+    exposurePointFeature.setValue(null);
+
+    exposurePointFeature.updateBuilder(mockCaptureRequestBuilder);
+
+    ArgumentCaptor<MeteringRectangle[]> meteringRectangleCaptor =
+        ArgumentCaptor.forClass(MeteringRectangle[].class);
+
+    verify(mockCaptureRequestBuilder, times(1))
+        .set(eq(CaptureRequest.CONTROL_AE_REGIONS), meteringRectangleCaptor.capture());
```

Comment:
Did you forget to test what `meteringRectangleCaptor.capture()` captured?

Reply:
My comment above could apply here as well to test this value.

Reply:
done